### ISSUE TITLE
Az.BotService is missing `/resources/webappv4.template.json`

### ISSUE THREAD

**Issue by stevengum** [Azure/azure-powershell#14218]
## Description
As of commit fcaddf8, `Az.BotService` doesn't include [`resources/`](https://github.com/Azure/azure-powershell/blob/fcaddf819ae36c045b1acbee67afb583095e7ba3/src/BotService/resources) or the ARM template [`webappv4.template.json`](https://github.com/Azure/azure-powershell/blob/fcaddf819ae36c045b1acbee67afb583095e7ba3/src/BotService/resources/webappv4.template.json). This missing template causes the `New-AzBotService` command to fail. The line looking for the template is below:

https://github.com/Azure/azure-powershell/blob/fcaddf819ae36c045b1acbee67afb583095e7ba3/src/BotService/custom/New-AzBotService.ps1#L225

## Steps to reproduce

```powershell
New-AzBotService -ResourceGroupName <your-resource-group-name> -Name <your-bot-handle> -ApplicationId <your-app-registration-id> -ApplicationSecret "<your-app-registration-secret>" -Location <your-bot-service-location> -Sku S1 -Description "<your-bot-description>" -Webapp

```

## Module versions

`master` & [Az.BotService 0.1.0](https://www.powershellgallery.com/packages/Az.BotService/0.1.0)

## Error

```
| Cannot retrieve the dynamic parameters for the cmdlet. Cannot find path 'C:\Users\<>\Documents\PowerShell\Modules\Az.BotService\0.1.0\resources\webappv4.template.json' because
| it does not exist.
```

@mikefrobbins FYI

Crosslinking https://github.com/MicrosoftDocs/bot-docs-pr/pull/2588. @JonathanFingold FYI

**Comment by dingmeng-xue** [comment]
Those files https://github.com/Azure/azure-powershell/tree/generation/src/BotService/resources were missed before publishing. 

@wyunchi-ms , please fix it.

**Comment by wyunchi-ms** [comment]
Fixed in PR #14221
